2022/workshop-capacity/workshop-capacity.md

```diff
@@ -0,0 +1,45 @@
+## Task Force Charter: Workshop Capacity
+
+Approval date: Monday, 17 October, 2022  
+Completion date: Friday, 28 October, 2022
+
+### Context
+We have been experiencing very limited availability of Instructors to teach Centrally-Organised Workshops (COW), and are currently unable
+to meet demand from Members and a-la-carte workshop requests. We have had to cancel or postpone several workshops. Some strategies are
+currently being explored to try to increase Instructor capacity, including providing compensation for Instructors and returning to in-person
+workshops, however, it remains to be seen whether these changes will result in increased capacity and it may be some time before we know if
+they are having the desired effects. 
+
+### Objective
+The objective of this task force is to evaluate three possible medium-term solutions to the problem of low Instructor availability and 
+provide a recommendation to the Executive Team as to which solution to pursue. The recommended solution should be able to be implemented quickly 
+(immediately to within 2 months) and should be sustainable for up to one year. The three solutions to explore are:
+1. Hiring a new Core Team member for one year who will have a significant portion of their time dedicated to teaching workshops, 
+2. Utilizing Core Team members to teach workshops, and
+3. Reducing our workshop offerings to match Instructor availability and communicating explicitly with Members about reduced capacity. 
```

Comment:
The objective of this task force is to identify a solution to the problem of low Instructor availability and provide a recommendation to the Executive Team. The recommended solution will be implemented quickly (immediately to within 2 months) and should be sustainable for up to one year. Two potential solutions to explore are provided below, however, the task force should brainstorm and weigh the pros and cons of all potential solutions:

- Utilizing Core Team members to teach workshops, and
- Reducing our workshop offerings to match Instructor availability and communicating explicitly with Members about reduced capacity.